packages/core-react/src/components/Card/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+
+export default function Card({ heading, body, img, row }) {
+  return (
+    <div className={clsx('ray-card', { 'ray-card--row': row })}>
```

Comment:
will need to add link support
```suggestion
    <div className={clsx('ray-card', { 'ray-card--row': row, 'ray-card--link': link })}>
```

Reply:
Good point

Reply:
I think I might do that in a follow-up since this pr is mainly about getting the infra off the ground